python/ray/util/client/server/proxier.py

```diff
@@ -191,13 +196,19 @@ def start_specific_server(self, client_id: str,
 
         serialized_runtime_env = job_config.get_serialized_runtime_env()
 
+        node = self._get_node()
```

Comment:
why do we need this and not directly return _session_dir?

Reply:
Because we also call `get_log_file_handles`